src/Hidden/examples.stories.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import {Meta} from '@storybook/react'
+import {Button, Link, Text, StateLabel, BranchName, Box} from '..'
+import {ArrowRightIcon} from '@primer/octicons-react'
+
+import {PageHeader} from '../PageHeader'
+import Hidden from '../Hidden'
+
+const meta: Meta = {
+  title: 'Drafts/Components/Hidden/Examples',
+  parameters: {
+    layout: 'fullscreen',
+    controls: {expanded: true},
+  },
+  args: {},
+}
+
+// See if there is an interest to take this into global params
+
+const PrimerViewports = {
```

Comment:
Similar to the shared note above, would be awesome if we could derive this from a shared spot to keep everything in sync (and pull from when primitives are ready as Mike mentions)

Reply:
I was actually thinking to move them over to the global storybook parameters. Because the current preview screen options are `['Small mobile', 'Large mobile', 'Tablet']` and they don't mean much (at least to me) - and what breakpoints they use are not explicit either. I think it would be more useful to use the breakpoints we use in Primer. What do you think? cc @mperrotti as well as he mention below

I pushed a [commit](https://github.com/primer/react/pull/2729/commits/41e4b635943dd90384dbbc4b16caec78d88ecf26) for this change. Let me know what you think 🙌🏻 

<img width="546" alt="Story book responsive screen option list that includes values 'XSmall', 'Small', 'Medium', 'Large', 'XLarge', 'XXLarge'" src="https://user-images.githubusercontent.com/1446503/212581477-ed974a01-6022-4d13-9cb2-f23d6e2fcac4.png">